client/src/components/scheduled-events/Event.vue

```diff
@@ -158,6 +202,7 @@ export default {
             line-height: 20px;
             letter-spacing: 0.25px;
             color: #2c2c2c;
```

Comment:
check if you can yse variable from the theme

Reply:
I check this when created layout - don't have variables for this colors